### DISCUSSION TITLE
An application configuration change has been detected - and I dont know why

### DISCUSSION THREAD

**Discussion by cholzer79** [abraunegg/onedrive#2887] [Q&A] [answered]
Today I deployed the docker version of onedrive _(driveone/onedrive:edge also tried driveone/onedrive:lastet)_ using Portainer.
The only thing I use the config file for is to configure the webhook (can I use the ENV to configure that instead of the conf?).
The initial resync worked great! All files were downloaded, webhoook works, changes got pushed correctly and quickly.

Then I stopped the onedrive container, set the ONEDRIVE_RESYNC ENV to 0 and launched the container again.

Now i get this:

```
usermod: no changes
# Running container as user: onedrive
# Base Args: --monitor
# Changing ownership permissions on /onedrive/data and /onedrive/conf to onedrive:onedrive
# Launching 'onedrive' as onedrive via gosu
Reading configuration file: /onedrive/conf/config
Configuration file successfully loaded
An application configuration change has been detected where a --resync is required
```

Is it possible that this is caused by the _Changing ownership permissions_ step? 
I have noticed that the modification date of the conf file was changed when the container started (or is restarted).

I am using binds.
![Screenshot 2024-10-06 193848](https://github.com/user-attachments/assets/6556f831-75d5-461f-99ed-6a91c0d163c8)

_/edit_
in case someone else comes across this, adding the host path  from the data bind to the conf file as sync_dir fixes the problem. :)
https://github.com/abraunegg/onedrive/discussions/2839#discussioncomment-10795424

**Comment by abraunegg** [comment, marked as answer]
@cholzer79 
Please raise a bug report

**Reply by cholzer79** [reply]
Done. Not sure if I did it correctly as some parts of the template dont quite seem to apply to the docker version of onedrive.
https://github.com/abraunegg/onedrive/issues/2889
